README.md

```diff
@@ -1,2 +1,70 @@
-# .github
-Organization-wide repository for common governance documents.
+# Notary Project Overview
+
+The Notary Project is a set of specifications and tools intended to provide a cross-industry standard for securing software supply chains through image signing and verification, signature portability, and key management. 
+
+The Notary Project started with an implementation for signing images in container registries and ensuring their integrity before deployment. The initial implementation uses [The Update Framework (TUF)](https://github.com/theupdateframework) and requires registries to host additional server infrastructure for managing signing keys and image metadata. This server infrastrcture tightly integrates with the container registry and keeps track of the images pushed to the registry. There is also a client component in the form of the `notary` command line interface (CLI) that can be used by developers or CI/CD pipelines to sign and push container images and update the metadata. The CLI wraps the communication to the registry as well as to the key and metadata management server component. The most prominent use of this implementation is in Docker Content Trust (DCT). The server and the client implementation can be found in the [notary](https://github.com/notaryproject/notary) repository under the Notary Project organization.
+
+Due to the tight integration between the registry and the key and metadata management server component, portability of signatures between registries is limited. This also limits the portability of the container images. Also, container registry APIs are getting standardized under the [Open Containers Initiative (OCI)](https://github.com/opencontainers) and new capabilities are added. To overcome the portability challenges and enable future flexibility and standardization, the Notary Project community decided to concentrate on specifications for software supply chains and provide reference implementations.
```

Comment:
```suggestion
Due to the tight integration between the registry, key, and metadata management server components, the portability of signatures between registries is limited which also limits the portability of the container images. Container registry APIs have begun standardizing on the [Open Containers Initiative (OCI)](https://github.com/opencontainers) and with new capabilities added. To overcome portability challenges and enable future flexibility and standardization, the Notary Project community decided to concentrate on specifications for software supply chains and provide reference implementations.
```

Reply:
Link to where the decision for this was recorded/captured or provide a footnote detailing this event.

Reply:
Updated the text. Looking through the notes to find the ones that document the discussion.

Reply:
that works, thank you!